docs/reference/predicates.md

```diff
@@ -223,6 +223,15 @@ route1: Path("/test") -> "http://www.zalando.de";
 route2: Path("/test") && False() -> "http://www.github.com";
 ```
 
+## Shutdown
+
+Evaluates to true if Skipper is shutting down. Can be used to create customized healthcheck.
+
+```
+health_up: Path("/health") -> inlineContent("OK") -> <shunt>;
+health_down: Path("/health") && Shutdown() -> status(503) -> inlineContent("shutdown") -> <shunt>;
```

Comment:
I think it's a great idea, composable and usable in all kind of dataclients.

Reply:
I really like the design as well. It seems to be elegant and pragmatic.